examples/3d/atmosphere.rs

```diff
@@ -0,0 +1,110 @@
+//! This example showcases pbr atmospheric scattering
+
+use std::f32::consts::PI;
+
+use bevy::{
+    core_pipeline::tonemapping::Tonemapping,
+    pbr::{Atmosphere, AtmosphereSettings, CascadeShadowConfigBuilder},
+    prelude::*,
+};
+use light_consts::lux;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, (setup_camera_fog, setup_terrain_scene))
+        .add_systems(Update, dynamic_scene)
+        .run();
+}
+
+fn setup_camera_fog(mut commands: Commands) {
+    commands.spawn((
+        Camera3d::default(),
+        Camera {
```

Comment:
Setting the exposure lower at noon and higher at sunset would make the showcase look more balanced, as well as adding a secondary directional light (moon) at night time to showcase a day-night cycle.